include/bar/bar.h

```diff
@@ -32,6 +32,9 @@ struct workspace {
 	bool urgent;
 };
 
+/** Global bar state */
+extern struct bar swaybar;
```

Comment:
No need to have a global struct config since swaybar which points to it was already global.